packages/compass-web/src/entrypoint.tsx

```diff
@@ -287,54 +288,56 @@ const CompassWeb = ({
           {...LINK_PROPS}
         >
           <PreferencesProvider value={preferencesAccess.current}>
-            <LoggerAndTelemetryProvider value={loggerAndTelemetry}>
-              <WithAtlasProviders>
-                <AtlasCloudConnectionStorageProvider
-                  orgId={orgId}
-                  projectId={projectId}
-                  autoConnectConnectionId={autoConnectConnectionId}
-                >
-                  <ConnectionsManagerProvider
-                    value={connectionsManager.current}
+            <LoggerProvider value={logger}>
+              <TelemetryProvider value={{ createTrack: () => logger.track }}>
```

Comment:
If we're splitting them in compass, we should also split the compass-web implementation, but here specifically, because the only required behavior from compass-web is to propagate tracking events to parent component, this can be simplified to something like this:

```suggestion
              <TelemetryProvider value={{ createTrack: () => (...args) =>  onTrackRef.current?.(...args)}}>
```

And then the only thing you need to do for logging-and-telemetry is clean up unused track code